src/app/features/room/RoomInput.tsx

```diff
@@ -318,7 +319,9 @@ export const RoomInput = forwardRef<HTMLDivElement, RoomInputProps>(
           content['m.relates_to'].is_falling_back = false;
         }
       }
-      mx.sendMessage(roomId, content);
+      if (!uploadBoardHandlers.current) {
+          mx.sendMessage(roomId, content);
+      }
```

Comment:
It will be a good idea if we can extract the `content` generation process in separate function so it can be used here and for caption.

Reply:
This code was reverted.